### ISSUE TITLE
Add a "tripwire" test to Dart SDK to detect breakages

### ISSUE THREAD

**Issue by alorenzen** [dart-lang/angular#1415]
Tracking issue for follow-on action for https://github.com/dart-lang/angular/issues/1406 breakage.

We should add a simple angular app to the Dart SDK test suite (i.e. golem) that can detect breakages or major regressions in either a presubmit or postsubmit check.

**Comment by natebosch** [comment]
yes, we do plan on getting a small `build_runner` test suite running in golem. We have discussed it in the past but I can't find an existing tracking issue so I filed https://github.com/dart-lang/build/issues/1639
